vscode-trace-extension/package.json

```diff
@@ -332,10 +332,10 @@
         "jsonc-parser": "^3.3.1",
         "lodash": "^4.17.15",
         "terser": "4.8.1",
-        "traceviewer-base": "^0.9.1",
-        "traceviewer-react-components": "^0.9.1",
+        "traceviewer-base": "0.10.0",
```

Comment:
I notice the "^" is gone. Since the libraries are local, I think it makes no practical difference.